plugins/logging/logging.cpp

```diff
@@ -117,3 +122,52 @@ LOGGING_API unsigned int LOGGING_CALL logGetElapsedMs(ICodeContext *ctx)
 {
     return ctx->getElapsedMs();
 }
+
+LOGGING_API char * LOGGING_CALL getTraceID(ICodeContext *ctx)
+{
+    StringBuffer ret;
+    Owned<IProperties> httpHeaders = ctx->queryContextLogger().getSpanContext();
+    if (httpHeaders)
```

Comment:
Can this ever return null?  If not it is clearer without the if() test.  (It also confuses static analysers).